lib/src/macos_app.dart

```diff
@@ -301,6 +301,13 @@ class _MacosAppState extends State<MacosApp> {
     return result;
   }
 
+  Iterable<LocalizationsDelegate<dynamic>> get _localizationsDelegates sync* {
```

Comment:
`MacosLocalizations` and `DefaultMacosLocalizations` should be created. The back button should have it as its semantic label

Reply:
Not sure what mac-specific localizations are needed. We can work that in later, current localizations seem to work fine